c/xcsp/README.md

```diff
@@ -1,15 +1,17 @@
-This programs are obtained by converting the benchmarks provided for the [XCSP_to_C tool][1] to C-programs.
+<!--
+This file is part of the SV-Benchmarks collection of verification tasks:
+https://github.com/sosy-lab/sv-benchmarks
+
+SPDX-FileCopyrightText: 2020 Dirk Beyer <dirk.beyer@sosy-lab.org>
```

Comment:
I think this should refer to "The SV-Benchmarks Community" ([docs](https://github.com/sosy-lab/sv-benchmarks/blob/master/CONTRIBUTING.md#license)).

Reply:
Can you point me closer to what you want to reference with that link?
I've read the doc before, but I don't find anything about crediting the community.

Reply:
Sorry, I misremembered the content of this link without checking. Indeed, only the discussion in #1099 would tackle this topic so far.

Reply:
No worries, it should be adjusted now